Python/Product/VSInterpreters/CPythonInterpreterFactoryProvider.cs

```diff
@@ -32,7 +32,7 @@ class CPythonInterpreterFactoryProvider : IPythonInterpreterFactoryProvider {
         private readonly Dictionary<string, PythonInterpreterInformation> _factories = new Dictionary<string, PythonInterpreterInformation>();
         const string PythonPath = "Software\\Python";
         internal const string FactoryProviderName = "Global";
-        private bool _watchRegistry;
+        internal bool _watchRegistry;
```

Comment:
is the change to internal needed? the PR doesn't seem to add any use of the member